maestro-sdk.cabal

```diff
@@ -20,65 +21,101 @@ source-repository head
 common common 
     ghc-options: -Wall
     default-extensions:
+      GADTs
       DataKinds
       DeriveGeneric
       DerivingStrategies
       DerivingVia
       GeneralisedNewtypeDeriving
+      FlexibleInstances
+      FlexibleContexts
       MultiParamTypeClasses
       NumericUnderscores
       OverloadedStrings
       RecordWildCards
       RoleAnnotations
       ScopedTypeVariables
       TemplateHaskell
+      TypeApplications
+      TypeFamilies
       TypeOperators
       QuasiQuotes
 
 library
     import:           common
     exposed-modules:  
-      Maestro.API
-      Maestro.API.Accounts
-      Maestro.API.Address
-      Maestro.API.Assets
-      Maestro.API.Datum
-      Maestro.API.Epochs
-      Maestro.API.General
-      Maestro.API.Pool
-      Maestro.API.Scripts
-      Maestro.API.Transaction
-      Maestro.API.TxManager
+      Maestro.API.V0
+      Maestro.API.V0.Accounts
+      Maestro.API.V0.Address
+      Maestro.API.V0.Assets
+      Maestro.API.V0.Datum
+      Maestro.API.V0.Epochs
+      Maestro.API.V0.General
+      Maestro.API.V0.Pool
+      Maestro.API.V0.Scripts
+      Maestro.API.V0.Transaction
+      Maestro.API.V0.TxManager
+
+      Maestro.API.V1
+      Maestro.API.V1.Addresses
+      Maestro.API.V1.Datum
+      Maestro.API.V1.General
+      Maestro.API.V1.Pools
+      Maestro.API.V1.Transactions
+      Maestro.API.V1.TxManager
 
-      Maestro.Client
-      Maestro.Client.Core
-      Maestro.Client.Core.Pagination
-      Maestro.Client.Datum
       Maestro.Client.Env
-      Maestro.Client.Epochs
-      Maestro.Client.Accounts
-      Maestro.Client.Address
-      Maestro.Client.Assets
-      Maestro.Client.General
-      Maestro.Client.Pools
-      Maestro.Client.Scripts
-      Maestro.Client.Transaction
-      Maestro.Client.TxManager
+      Maestro.Client.Error
+      Maestro.Client.V0
+      Maestro.Client.V0.Core
+      Maestro.Client.V0.Core.Pagination
+      Maestro.Client.V0.Datum
+      Maestro.Client.V0.Epochs
+      Maestro.Client.V0.Accounts
+      Maestro.Client.V0.Address
+      Maestro.Client.V0.Assets
+      Maestro.Client.V0.General
+      Maestro.Client.V0.Pools
+      Maestro.Client.V0.Scripts
+      Maestro.Client.V0.Transaction
+      Maestro.Client.V0.TxManager
+
+      Maestro.Client.V1
+      Maestro.Client.V1.Core
+      Maestro.Client.V1.Core.Pagination
+      Maestro.Client.V1.Addresses
+      Maestro.Client.V1.Datum
+      Maestro.Client.V1.General
+      Maestro.Client.V1.Pools
+      Maestro.Client.V1.Transactions
+      Maestro.Client.V1.TxManager
```

Comment:
Please add support for `/txmanager/turbosubmit`: https://docs-v1.gomaestro.org/docs/API%20reference/Transaction%20Manager/tx-manager-turbo-submit
This might help with the utxo issues you're encountering at GY.